Makefile

```diff
@@ -36,6 +36,7 @@ SRC_FILES = main.cpp \
 			Location.cpp \
 			ServerData.cpp \
 			cgi.cpp \
+			outputLogs.cpp 
```

Comment:
add a " \" in the end of the line so when you need to add a new file you won't edit two lines